Test failed ov-keymap{1,2,3}

@alphapapa.
Currently, ov-test/ov-keymap{1,2,3} failed. How this issue solve?

### Regenerate steps
1. clone this repository.
2. do `cask install`.
3. do `make test`.
4. see error like below

```
Ran 28 tests in 0.644 seconds
3 unexpected results:
   FAILED  ov-test/ov-keymap1
   FAILED  ov-test/ov-keymap2
   FAILED  ov-test/ov-keymap3
make[1]: *** [Makefile:13: unit] Error 1
make[1]: Leaving directory '/home/conao/dev/forks/ov'
make: *** [Makefile:7: test] Error 2
```

--- a/test/ov-test.el
+++ b/test/ov-test.el
@@ -257,7 +257,7 @@
   (should (ov? (car ov1)))
   (should (eq 8 (length (ov-all))))
   (ov-goto-next)
-  (execute-kbd-macro (read-kbd-macro "C-n"))
+  (execute-kbd-macro "C-n")
   (should (eq 0 (length (ov-all)))))
 
 (ert-deftest ov-test/ov-keymap2 ()
@@ -270,7 +270,7 @@
   (should (ov? ov1))
   (should (eq (point-at-bol) (ov-beg ov1)))
   (should (eq (point-at-eol) (1- (ov-end ov1))))
-  (execute-kbd-macro (read-kbd-macro "C-p"))
+  (execute-kbd-macro "C-p")
   (should (eq 0 (length (ov-all)))))
 
 (ert-deftest ov-test/ov-keymap3 ()
@@ -281,7 +281,7 @@
   (should (eq (point-min) (ov-beg ov1)))
   (should (eq (point-max) (ov-end ov1)))
   (should (ov? (ov-at)))
-  (execute-kbd-macro (read-kbd-macro "C-p"))
+  (execute-kbd-macro "C-p")
   (should-not (ov? (ov-at))))
 
 (ert-deftest ov-test/ov-read-only1 ()

